src/lib/ngsiNotify/Notifier.cpp

```diff
@@ -104,12 +104,11 @@ void Notifier::sendNotifyContextRequest(NotifyContextRequest* ncr, const std::st
     int          port;
     std::string  uriPath;
     std::string  protocol;
-    
+    std::string  initialUrl = url;
```

Comment:
OK,

parseUrl calls stringSplit that does:

    int stringSplit(const std::string& in, char delimiter, std::vector<std::string>& outV)
    {
      char* s          = strdup(in.c_str());

There is my copy.
initialUrl to be removed.